terraform/modules/tf_stream_alert_flow_logs/main.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_flow_log" "vpc_flow_log" {
+  count           = "${length(var.targets["vpcs"])}"
+  vpc_id          = "${element(var.targets["vpcs"],count.index)}"
+  log_group_name  = "${aws_cloudwatch_log_group.flow_log_group.name}"
+  iam_role_arn    = "${aws_iam_role.flow_log_role.arn}"
+  traffic_type    = "ALL"
+}
+
+resource "aws_flow_log" "subnet_flow_log" {
+  count           = "${length(var.targets["subnets"])}"
+  subnet_id       = "${element(var.targets["subnets"],count.index)}"
+  log_group_name  = "${aws_cloudwatch_log_group.flow_log_group.name}"
+  iam_role_arn    = "${aws_iam_role.flow_log_role.arn}"
+  traffic_type    = "ALL"
```

Comment:
traffic type can be `ALL` `ACCEPT`, or `REJECT` -- i can definitely make this configurable